src/public/modules/forms/admin/directiveViews/configure-mobile.client.view.html

```diff
@@ -1,15 +1,22 @@
 <div class="row">
-  <div class="toggle-option">
+  <div class="toggle-option label-bottom">
     <div
       ng-class="{'toggle-option--disabled' : field.hasAdminExceededSmsLimit || field.hasRetrievalError}"
-      class="col-xs-8 label-custom label-medium label-bottom"
+      class="col-xs-8 justify-content-start"
```

Comment:
Can you double check if `justify-content-start` does anything? We are still using bootstrap 3, I don't see any documentation on this css

Reply:
I referenced the newer version of bootstrap [here](https://getbootstrap.com/docs/4.0/layout/grid/#horizontal-alignment) where it aligned horizontally as I wasn't aware of the exact version of bootstrap used. 

Will double check if this does anything

Reply:
The current version of bootstrap being used can be found in our package.json

Reply:
checked; this is extra, will remove